test/functional/vimscript/executable_spec.lua

```diff
@@ -17,6 +17,21 @@ describe('executable()', function()
     eq(1, call('executable', 'false'))
   end)
 
+  if iswin() then
+    it('exepath respects shellslash', function()
+      command('let $PATH = fnamemodify("./test/functional/fixtures/bin", ":p")')
+      eq('test\\functional\\fixtures\\bin\\null.CMD', call('fnamemodify', call('exepath', 'null'), ':.'))
+      command('set ssl')
```

Comment:
in scripts and code it is almost always preferred to spell out the full name (helps with `grep` etc)
```suggestion
      command('set shellslash')
```